projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.html

```diff
@@ -301,12 +301,22 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
       </cx-fs-accordion-item>
     </ng-container>
   </div>
-  <div class="mt-5">
-    <button
-      class="m-auto col-6 btn-primary btn-block text-uppercase"
-      (click)="retrieveQuote(quote)"
-    >
-      {{ 'fscommon.retrieve' | cxTranslate }}
-    </button>
+  <div class="row mt-5 d-flex justify-content-center">
+    <div class="col-xs-12 col-sm-5 mb-3 mb-md-0">
```

Comment:
`col-xs-12` should be `col-12`, `xs` is depricated in Bootstrap 4.
No need for `mb-3 mb-md-0`. You can just set `mt-3` on the right botton.